storage/google/cloud/storage/client.py

```diff
@@ -153,6 +153,26 @@ def _pop_batch(self):
         """
         return self._batch_stack.pop()
 
+    def _use_or_init_bucket(self, bucket_or_name):
```

Comment:
Made a helper - for now it's used in 3 places

Reply:
Nit on naming: how about `_bucket_arg_to_bucket` to align with the similar helper in BigQuery?

https://github.com/googleapis/google-cloud-python/blob/0e231cab5f066ae5b3e254d0cb7d204e3dac7ebc/bigquery/google/cloud/bigquery/table.py#L1771

Also, this might be well suited for the `bucket.py` module, similar to BigQuery, as I expect the `client.py` file will expand and the `bucket.py` file will shrink as we move implementation over to get ready for deprecation.

Reply:
No problem, renamed